packages/lambda-tiler/src/routes/version.ts

```diff
@@ -1,6 +1,16 @@
-import { HttpHeader, LambdaHttpResponse } from '@linzjs/lambda';
+import { BasemapsConfigProvider, ConfigProviderMemory } from '@basemaps/config';
+import { HttpHeader, LambdaHttpRequest, LambdaHttpResponse } from '@linzjs/lambda';
 
-export function versionGet(): Promise<LambdaHttpResponse> {
+import { ConfigLoader } from '../util/config.loader.js';
+
+function getConfigHash(cfg: BasemapsConfigProvider): { id: string; hash?: string } | undefined {
+  if (!ConfigProviderMemory.is(cfg)) return undefined;
+  if (cfg.id == null) return undefined;
```

Comment:
We still require dynamoDB to points to the config hash here, just double check our final goal is to use environment variables for the config hash? If so we might not need the id here?

Reply:
the `id` for JSON files is the config bundle id eg `cb_01123abc123` 

generally this should be a config from JSON file by default.

Reply:
Oh, I forgot that's the json id as well. I only thought that's the dynameDB id as it starts with `cb_`.